sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaProperties.cs

```diff
@@ -35,7 +35,7 @@ internal SchemaProperties(string content, string location, SerializationType xSc
         /// <summary>
         /// Serialization type for the schema being stored.
         /// </summary>
-        internal SerializationType Type { get; }
+        public SerializationType Type { get; }
```

Comment:
interesting, do you remember why was it internal before?

Reply:
I don't know why it was internal before.